tests/integration/validate/test_validate_command.py

```diff
@@ -142,7 +142,8 @@ def test_lint_supported_runtimes(self):
             "nodejs18.x",
             "provided",
             "provided.al2",
-            "provided.al2023",
+            # Note(hnnasit): provided.al2023 is not supported by cfn-lint yet. Uncomment once it is supported.
+            # "provided.al2023",
```

Comment:
I saw that `cfn-lint` has open PR to add `provided.al2023` into their list: https://github.com/aws-cloudformation/cfn-lint/pull/2943/files

I wonder we can leave it there for now and accept failures until it gets updated 🤔 Or add it here conditionally by checking the library version.

Reply:
oh didn't see that. I think we can let the test fail for now since it throws a user exception and once the `cfn-lint` PR gets merged in, the test should be fixed automatically.